### PR TITLE
adapt public rest api to cf style

### PR DESCRIPTION
**Pull request by qibobo**
[#159391488]

### CODE REVIEW

File: api/lib/routes/routeHelper.js [hunk 1]
```diff
@@ -13,19 +13,32 @@ exports.isString = function(value) {
...
+exports.pagination = function(resultList, page, resultsPerPage, req) {
   var result = {};
   var totalResults = resultList.length;
   var totalPages = 0;
   var perPage = Number.parseInt(resultsPerPage);
   var pageNo = Number.parseInt(page);
-  totalPages = (totalResults % perPage) === 0 ? (totalResults / perPage) : (totalResults / perPage + 1);
+  var prevUrl = null;
+  var nextUrl = null;
+  totalPages = (totalResults % perPage) === 0 ? (totalResults / perPage) : (Number.parseInt(totalResults / perPage) + 1);
```
Comment (boyang9527) [inline api/lib/routes/routeHelper.js]: why Number.parseInt ?

Comment (qibobo) [reply api/lib/routes/routeHelper.js]: var totalResults = 5;
var resultsPerPage = 2;
var totalPages = 0;
var perPage = Number.parseInt(resultsPerPage);
totalPages = (totalResults % perPage) === 0 ? (totalResults / perPage) : (totalResults / perPage + 1);
//totalPages = (totalResults % perPage) === 0 ? (totalResults / perPage) : (Number.parseInt(totalResults / perPage) + 1);
console.log(totalPages)
Without Number.parseInt the result of the code above is 3.5 and after adding it the result is 3.

Comment (boyang9527) [reply api/lib/routes/routeHelper.js]: thanks for the explanation.

**Comment by boyang9527** [comment]
LGTM

File: docs/Public_API.rst [hunk 2]
```diff
@@ -30,7 +30,7 @@ Parameters
 +--------------------+-------------------------------------------------------------------------------+---------------------------------------------------------------------+-----------------------+----------------------------------+
 | end-time           | The end time                                                                  | int, the number of nanoseconds elapsed since January 1, 1970 UTC.   | false. default 'now'  | end-time=1494989549117047288     |
 +--------------------+-------------------------------------------------------------------------------+---------------------------------------------------------------------+-----------------------+----------------------------------+
-| order              | The order type. The scaling history will be order by timestamp asc or desc.   | string,”asc” or “desc”                                              | false. default desc   | order=desc                       |
+| order-direction    | The order type. The scaling history will be order by timestamp asc or desc.   | string,”asc” or “desc”                                              | false. default desc   | order=desc                       |
```
Comment (cdlliuy) [inline docs/Public_API.rst]: missed "pre_url" and "next_url" in this table

Comment (cdlliuy) [inline docs/Public_API.rst]: use " instead of ” ?

Comment (cdlliuy) [reply docs/Public_API.rst]: order-direction=desc

File: api/lib/routes/aggregated_metrics.js [hunk 3]
```diff
@@ -13,13 +13,13 @@ module.exports = function(settings) {
     var metricType = req.params.metric_type;
     var startTime = req.query["start-time"];
     var endTime = req.query["end-time"];
-    var order = req.query["order"];
+    var orderDirection = req.query["order-direction"];
```
Comment (cdlliuy) [inline api/lib/routes/aggregated_metrics.js]: considering the backward compatibility  , could we support both "order" and "order-direction" as a valid parameter?  In the external document, we can claim the "order-direction" only.

**Comment by cdlliuy** [comment]
LGTM

**Pull request merged**